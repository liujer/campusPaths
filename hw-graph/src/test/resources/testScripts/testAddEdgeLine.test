# Tests adding of edges by using two nodes
# that are in a line
CreateGraph C
AddNode C one
AddNode C two
AddEdge C one two e12
ListNodes C
ListChildren C one
ListChildren C two