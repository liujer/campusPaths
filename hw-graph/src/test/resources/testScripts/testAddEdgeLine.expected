# Tests adding of edges by using two nodes
# that are in a line
Created graph C
Added node one to C
Added node two to C
Added edge e12 to C
ListNodes of C output: one two
the children of one in C are: two(e12)
the children of two in C are: